PlantPlaces23FS003/Pages/Index.cshtml

```diff
@@ -5,13 +5,13 @@
     ViewData["Title"] = "Home page";
     var brand = ViewData["Brand"];
     List<Specimen> specimens = (List<Specimen>) ViewData["Specimens"];
-    var message = ViewData["Message"];
+    var message = ViewData["WeatherMessage"];
```

Comment:
I changed this because variable names should be descriptive.
https://dev.to/tanisha03/clean-code-part-1-meaningful-names-4khp